distributor/distributor.go

```diff
@@ -0,0 +1,13 @@
+package distributor
+
+import cm "scootdev/scoot/clustermembership"
+
+/*
+ * Interface for choosing a node in a cluster to send
+ * work to.  When DistributeWork is called a node from
+ * the passed in cluster should be returned, where work
+ * can be scheduled.
+ */
+type Distributor interface {
+	DistributeWork(cluster cm.Cluster) cm.Node
```

Comment:
This is a good intermediate stage.

I think this will grow in at least two ways:
a) take the piece of work (or at least some info about it) to distribute it meaningfully
b) take all the pieces of work in the queue to assign them at once

Do I have that right?